app/helpers/patient_query_helper.rb

```diff
@@ -580,7 +580,8 @@ def linelist(patients, workflow, tab)
       details[:reason_for_closure] = patient[:monitoring_reason] || '' if fields.include?(:reason_for_closure)
       details[:closed_at] = patient[:closed_at]&.rfc2822 || '' if fields.include?(:closed_at)
       details[:transferred_at] = patient[:latest_transfer_at]&.rfc2822 || '' if fields.include?(:transferred_at)
-      details[:latest_report] = patient[:latest_assessment_at]&.rfc2822 || '' if fields.include?(:latest_report)
+      latest_report = { timestamp: patient[:latest_assessment_at]&.rfc2822, symptomatic: patient.latest_assessment_symptomatic? }
```

Comment:
@holmesie I think it might be worth adding a `latest_report_symptomatic` field to `Patient` because loading assessments here one at a time or even all together with a join is going to significantly slow down the linelists

Reply:
I think that's a great idea

Reply:
sounds good, @hackrm is this something you're comfortable/familiar with doing or would you like me to handle it? I can also try to walk you through it

Reply:
@timwongj , I have yest to make this type of change, so I would appreciate the chance to walk through it with you! I'll follow up with you in a DM.

Reply:
This has been addressed as of commit [CC62FB1](https://github.com/SaraAlert/SaraAlert/pull/745/commits/cc62fb1a262d8f3b8ed8f251fe69d56bf27ce40d)